recipes/protobuf/all/conandata.yml

```diff
@@ -21,6 +24,11 @@ sources:
     url: "https://github.com/protocolbuffers/protobuf/archive/refs/tags/v3.17.1.tar.gz"
     sha256: "036d66d6eec216160dd898cfb162e9d82c1904627642667cc32b104d407bb411"
 patches:
+  "3.21.12":
+    - patch_file: "patches/protobuf-3.21.12-upstream-macos-macros.patch"
+      patch_description: "Handle case where macOS SDK macros may conflict with protobuf message types"
+      patch_type: "bugfix"
+      patch_source: "https://github.com/protocolbuffers/protobuf/pull/10103"
```

Comment:
I don't see the connection of the patch with https://github.com/protocolbuffers/protobuf/pull/10103, that PR was merged a year ago, are you sure this is the correct link? Could you please elaborate a bit more?

Reply:
I did not write the patch, I only copied it like everyone else before me did. Although committed to `master` the patch is not part of the `21.x` branch of protobuf, yet: [code](https://github.com/protocolbuffers/protobuf/blob/21.x/src/google/protobuf/port_def.inc#L870-L873)

Reply:
c.f. #16453